src/pages/PrintCards.tsx

```diff
@@ -103,7 +145,10 @@ const PrintCards: React.FC<RouteComponentProps> = (props) => {
       </div>
       <div className="PrintCardsContainer">
         <div className="PrintCardsContent">
-          <ComponentToPrint ref={componentRef} />
+          <ComponentToPrint
+            ref={componentRef}
+            passAllDimensions={allDimensions}
+          />
         </div>
       </div>
       <div className="Footer">
```

Comment:
Can't see the code, but the back button should probably take you back to the course rather than the overall dashboard page. Might be better to put the courseID in the URL as a query param, like in the DisplayCards and Preview pages.

Reply:
Ok yeah, that increases the useability of the print cards page!